docs/backend/setup/common-errors-and-solutions.md

```diff
@@ -30,21 +30,18 @@ The engine "node" is incompatible with this module. Expected version "12.21.0".
 ```
 
 #### Solution
-The project uses node version `12.21.0`. Use `nodenv` to manage your node versions and easily switch between them.
+The project uses the node version specified in `.tool-versions`. Use `asdf` to manage your node versions and easily switch between them.
```

Comment:
Love this generalization to make the documentation more up-to-date and maintainable.